railties/lib/rails/application/configuration.rb

```diff
@@ -278,22 +278,30 @@ def load_defaults(target_version)
         @loaded_config_version = target_version
       end
 
-      def encoding=(value)
+      # :attr_writer: encoding
+
+      # Documentated as an attr_writer
+      def encoding=(value) # :nodoc:
         @encoding = value
         silence_warnings do
           Encoding.default_external = value
           Encoding.default_internal = value
         end
       end
 
+      # Configures the application to be API only. Sets the debugging
+      # information rendering to preserve the response format.
```

Comment:
Should this be `:attr_writer:` too?  Or were you handling it separately because it returns `:api` instead of `value`?